tests/test-privacy.php

```diff
@@ -75,6 +75,12 @@ public function test_addPrivacyPolicyContent() {
 	 */
 	public function test_namespace() {
 		$last_updated_before = get_blog_details()->last_updated;
+		// Doing it right
```

Comment:
Every-time we try to setup the wp privacy we need to be sure is being used in the admin area